mods/default/nodes.lua

```diff
@@ -435,7 +436,7 @@ minetest.register_node("default:dirt_with_grass", {
 		{name = "default_dirt.png^default_grass_side.png",
 			tileable_vertical = false}},
 	groups = {crumbly = 3, soil = 1, spreading_dirt_type = 1},
-	drop = 'default:dirt',
+	drop = "default:dirt",
```

Comment:
When i wrote something like:
'We are changing ' to " for nodenames'
I only meant do that in touched code, not for all nodes. I appreciate you doing this, but this PR is complex enough and shouldn't have any out-of-scope changes added.

Reply:
> I also threw in freebies, fixing missed ' > " for dropped node names (FYI I plan to fix the rest in a PR after this, there are a few throughout the code)

Best do all in a separate PR then.